docs/upgrade.html

```diff
@@ -18,6 +18,9 @@
 Users upgrading from versions prior to 0.10.1.x should follow the upgrade guide <a href="#upgrade_10_1">here</a>. Users upgrading from 0.10.1.0
 can upgrade the brokers one at a time: shut down the broker, update the code, and restart it.
 
+<p>Starting with version 0.10.2, the Kafka Java client is compatible with older brokers.  Version 0.10.2 clients can talk to version 0.10.0 or newer brokers.
+However, if your brokers are older than 0.10.0, you must upgrade the broker before rolling out new clients.
```

Comment:
Thinking about it, we should probably mention in the class javadoc of `KafkaConsumer` and `KafkaProducer` the behaviour with regards to older brokers.